testing/integration/kubernetes_agent_standalone_test.go

```diff
@@ -1109,35 +1111,8 @@ func k8sRenderKustomize(kustomizePath string) ([]byte, error) {
 }
 
 // generateESAPIKey generates an API key for the given Elasticsearch.
-func generateESAPIKey(esClient *elasticsearch.Client, keyName string) (string, error) {
```

Comment:
Removed in favor of `estools.APIKeyResponse`, the `elasticsearch` exporter requires the encoded version.